RNIap.podspec

```diff
@@ -4,14 +4,13 @@ package = JSON.parse(File.read(File.join(__dir__, 'package.json')))
 Pod::Spec.new do |s|
   s.name         = "RNIap"
   s.version      = package['version']
-  s.summary      = "RNIap"
-  s.description  = package['description']
+  s.summary	 = package['description']
   s.homepage     = "https://github.com/dooboolab/react-native-iap"
   s.license      = "MIT"
   s.author       = package['author']
-  s.platform     = :ios, "7.0"
-  s.source       = { :git => "https://github.com/dooboolab/react-native-iap", :tag => "v#{s.version}" }
-  s.source_files  = "ios/**/*.{h,m}"
+  s.platform     = :ios, "9.0"
+  s.source       = { :git => "https://github.com/dooboolab/react-native-iap", :tag => "#{s.version}" }
```

Comment:
Could you explain why you removed `v` before `#{s.version}`?

Reply:
![image](https://user-images.githubusercontent.com/26326015/63529375-82d32d00-c53f-11e9-9933-3c67481c144f.png)

This repository using version format as like '3.x.x' not 'v3.x.x'

Reply:
Thanks for the detail. I wasn't taking care so much on the release tags. :sweat_smile: